tests/unit/variant_browser.js

```diff
@@ -0,0 +1,99 @@
+/*
+ * European Variation Archive (EVA) - Open-access database of all types of genetic
+ * variation data from all species
+ *
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+var chai,expect = chai.expect, should = chai.should();
+
+describe('Most Severe ConsquenceType Tests', function(){
+    describe('getMostSevereConsequenceType() without any parameters', function(){
+        it('should return  empty array', function(){
+            expect(getMostSevereConsequenceType()).to.be.an('array').to.have.lengthOf(0);
+        });
+    });
+    describe('getMostSevereConsequenceType(values)', function(){
+        it('should not return emoty array', function(){
+            expect(getMostSevereConsequenceType(conseqData)).to.be.an('array').to.have.lengthOf(3);
+        });
+        it('should have missense_variant as first element in the array ', function(){
+            var consequenceType = getMostSevereConsequenceType(conseqData);
+            consequenceType[0].should.equal('missense_variant');
+        });
+    });
+
+});
+
+describe('Protein Substitution Score Tests', function(){
```

Comment:
The type of tests I had in mind was more like having:
```
testObjectBefore = {...}
expectedObjectAfter = {...}
expect(getProteinSubstitutionScore(testObjectBefore).to.deep.equal(expectedObjectAfter)
```
That test would also make easier to understand what those 3 functions do. It would be enough with a small object that has all the subarrays/subobjects at least once.